siege_utilities/geo/spatial_data.py

```
@@ -176,6 +176,164 @@ class SpatialDataError(RuntimeError):
+    def _get(url: str) -> Optional[bytes]:
+        try:
+            r = requests.get(url, timeout=timeout)
+            if r.status_code == 200:
+                return r.content
+            log.debug("census inventory crawl: %s → %s", url, r.status_code)
+        except Exception as exc:
+            log.debug("census inventory crawl: %s → %s", url, exc)
+        return None
+
+    inventory: dict = {
+        "updated_at": datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"),
```

Ruff (0.15.11)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)